README.md

```diff
@@ -73,34 +120,46 @@ It can be run in a couple of ways:
 1. Fire and forget
   * Execute on a single host with multiple cores (or 1 if that's all you have)
   * Some efficiency overhead as some steps aren't parallel
-2. Farm style
+1. Farm style
   * Requires 2 extra parameters in the initial command
   * See ``-help`` for further details
 
----
-
 ## Input
 
 ### Initial mapping
 
-BRASS has primarily been written to work with BWA mapped data.  You are likely to get the most useful output from BWA-mem.
+BRASS has primarily been written to work with BWA mapped data.  You are likely to get the most
+useful output from BWA-mem.
 
 ### Library quality
 
-Please be aware that paired-end libraries where properly-paired reads overlap are unlikely to produce good results.
+Please be aware that paired-end libraries where properly-paired reads are heavily overlapped are
+unlikely to produce good results.
 
 ### Additional mapping information
 
-BRASS requires accurate information regarding the insert size distribution and expects to find a ``*.bam.bas`` file
-co-located with the ``*.bam``'s.  These can be generated by the ``bam_stats`` program included in the
-[PCAP-core](https://github.com/ICGC-TCGA-PanCancer/PCAP-core) project.  If you use ``bwa_mem.pl`` to map your
-data (same repository) then this file is generated automatically for you.
+BRASS requires accurate information regarding the insert size distribution and expects to find a
+``*.bam.bas`` file co-located with the ``*.bam``'s.  These can be generated by the ``bam_stats``
+program included in the [PCAP-core](https://github.com/ICGC-TCGA-PanCancer/PCAP-core) project.
```

Comment:
Do we want to be linking to the cancerIT PCAP rather than the PanCan one?

Reply:
Good spot